### PR TITLE
export useful info in backtestResultExporter

### PR DESCRIPTION
**Pull request by hiyan**
Added some useful config info to export file. Should help if you review your tests results at later time.

### CODE REVIEW

File: plugins/backtestResultExporter.js [hunk 1]
```diff
@@ -89,6 +89,9 @@ BacktestResultExporter.prototype.processPerformanceReport = function(performance
 
 BacktestResultExporter.prototype.finalize = function(done) {
   const backtest = {
+    watch: config.watch,
```
Comment (askmike) [inline plugins/backtestResultExporter.js]: would you mind renaming this to `market`?

File: plugins/backtestResultExporter.js [hunk 2]
```diff
@@ -89,6 +89,9 @@ BacktestResultExporter.prototype.processPerformanceReport = function(performance
...
+    tradingAdvisor: config.tradingAdvisor,
+    strategy: config[config.tradingAdvisor.method],
```
Comment (askmike) [inline plugins/backtestResultExporter.js]: would you mind renaming this to `strategyParameters`?

**Review by askmike — COMMENTED** [review]
Great idea! I have something similar in my own code that I am planning to release soon, but I would love your contribution!

As part of a bigger renaming, would you mind updating the names above? Thanks!

File: plugins/backtestResultExporter.js (hunk 2 above)
Comment (hiyan) [reply plugins/backtestResultExporter.js]: sure :)

**Comment by askmike** [comment]
Woa, apologies for the late merge!

**Pull request merged**